lib/taste_tester/host.rb

```diff
@@ -40,7 +40,8 @@ def initialize(name, server)
 
     def runchef
       logger.warn("Running '#{TasteTester::Config.command}' on #{@name}")
```

Comment:
No idea, and I can't find a reference to this anywhere else either. Maybe @odcinek knows?